modules/serverless-install-kafka-odc.adoc

```diff
@@ -6,8 +6,41 @@
 [id="serverless-install-kafka-odc_{context}"]
 = Installing Knative Kafka
 
-The {ServerlessOperatorName} provides the Knative Kafka API that can be used to create a `KnativeKafka` custom resource:
+Knative Kafka provides integration options for you to use supported versions of the Apache Kafka message streaming platform with {ServerlessProductName}. Knative Kafka functionality is available in an {ServerlessProductName} installation if you have installed the `KnativeKafka` custom resource.
 
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} and Knative Eventing on your cluster.
+* You have access to a Red Hat AMQ Streams cluster.
+* Install the OpenShift CLI (`oc`) if you want to use the verification steps.
+
+// OCP
+ifdef::openshift-enterprise[]
+* You have cluster administrator permissions on {product-title}.
+endif::[]
+
+// OSD
+ifdef::openshift-dedicated[]
+* You have cluster or dedicated administrator permissions on {product-title}.
+endif::[]
+
+* You are logged in to the {product-title} web console.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Operators* -> *Installed Operators*.
+
+. Check that the *Project* dropdown at the top of the page is set to *Project: knative-eventing*.
+
+. In the list of *Provided APIs* for the {ServerlessOperatorName}, find the *Knative Kafka* box and click *Create Instance*.
+
+. Configure the *KnativeKafka* object in the *Create Knative Kafka* page.
++
+[IMPORTANT]
+====
+To use the Kafka channel, source, broker, or sink on your cluster, you must toggle the *enabled* switch for the options you want to use to *true*. These switches are set to *false* by default. Additionally, to use the Kafka channel, broker or sink, you must specify the bootstrap servers.
```

Comment:
> Additionally, to use the Kafka channel, broker or sink, you must specify the bootstrap servers.

For the broker: the "bootstrap" server can be overriden, with a ConfigMap that is in the "user namespace", and the broker referencing to that one (my original PR for the kafka broker doc had text for using the default global config, and a custom (or overriden) config/broker.


For the `kafka sink`, the bootstrap URL is only needed on that CR file - not on `Knative Kafka`

Reply:
@matzew can you open a follow up Jira for this please, it's out of scope for this PR which is just intended to get our docs in line with Jupiter requirements. This is previously existing content that was just moved slightly in this PR.